### PR TITLE
Refactor health monitor

### PR DESCRIPTION
**Pull request by patapufate**
This patch:

Clean Code tasks:
- Remove unnecessary boilerplate in CMakeList.txt and Package.xml.
- Add mantainer and license.
- Add Roslint and fix linter errors.
- Change folder structure for header files. Now implemented as "include/<package_name>/header_file.h".
- Change some variable names into camelCase style

Fix bug:
- Mismatch of fault_id variable type definition in ClearFault.msg and ReportFault.msg. It was changed the size of the error bitmask in the ClearFault.msg definition to be 64bit size.

It needs thorough testing, as I don't have the hardware at hand.

### CODE REVIEW

File: health_monitor/CMakeLists.txt [hunk 1]
```diff
@@ -175,45 +64,13 @@ target_include_directories(${NODE_APP_NAME} PUBLIC)
...
-
-## Add gtest based cpp test target and link libraries
-# catkin_add_gtest(${PROJECT_NAME}-test test/test_health_monitor.cpp)
-# if(TARGET ${PROJECT_NAME}-test)
-#   target_link_libraries(${PROJECT_NAME}-test ${PROJECT_NAME})
-# endif()
-
-## Add folders to be run by python nosetests
-# catkin_add_nosetests(test)
+)
```
Comment (hidmic) [inline health_monitor/CMakeLists.txt:76]: @patapufate nit: missing newline.

File: health_monitor/src/health_monitor.cpp [hunk 2]
```diff
@@ -159,12 +156,12 @@ void HealthMonitor::sendFaults()
     health_monitor::ReportFault message;
 
     faultArrayMutex.lock();
-    uint64_t temp_fault_id = faults;
+    uint64_t tempFaultIF = faults;
```
Comment (hidmic) [inline health_monitor/src/health_monitor.cpp:159]: @patapufate typo:
```suggestion
    uint64_t tempFaultID = faults;
```

**Review by hidmic — APPROVED** [review]
LGTM

File: health_monitor/CMakeLists.txt (hunk 1 above)
Comment (patapufate) [reply health_monitor/CMakeLists.txt]: Fixed

File: health_monitor/src/health_monitor.cpp (hunk 2 above)
Comment (patapufate) [reply health_monitor/src/health_monitor.cpp]: Fixed

**Pull request merged**